docs/config.rst

```diff
@@ -10,6 +10,9 @@ Configuration Options
 
         Paths (relative or absolute) to the source code that you wish to generate your API documentation from.
```

Comment:
Can we reword this section to make it clearer that we expect the package directory to be specified? Maybe mention implicit namespace packages as well.